examples/smallvil/input.rs

```diff
@@ -0,0 +1,139 @@
+use smithay::{
+    backend::input::{
+        Axis, AxisSource, ButtonState, Event, InputBackend, InputEvent, KeyboardKeyEvent, PointerAxisEvent,
+        PointerButtonEvent, PointerMotionAbsoluteEvent,
+    },
+    wayland::{
+        seat::{AxisFrame, ButtonEvent, FilterResult, MotionEvent},
+        SERIAL_COUNTER,
+    },
+};
+use wayland_server::{protocol::wl_pointer, Display};
+
+use crate::state::Smallvil;
+
+impl Smallvil {
+    pub fn process_input_event<I: InputBackend>(
+        &mut self,
+        display: &mut Display<Smallvil>,
+        event: InputEvent<I>,
+    ) {
+        match event {
+            InputEvent::Keyboard { event, .. } => {
+                let cx = &mut display.handle();
+                self.seat.get_keyboard().unwrap().input::<(), _>(
+                    cx,
+                    event.key_code(),
+                    event.state(),
+                    0.into(),
```

Comment:
What do these two zero values mean?

Reply:
Those were wrong, fixed to proper values